### PR TITLE
hack/boilerplate: added regex to strip Python shebang

### PR DESCRIPTION
**Pull request by falcon78921**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind failing-test

**What this PR does / why we need it**:

Fixes Prow Build Error:
`Boilerplate header is wrong for: /home/prow/go/src/k8s.io/kubernetes/hack/update_owners.py`

In regards to #79190

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:

N/A

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/boilerplate/boilerplate.py [hunk 1]
```diff
@@ -102,6 +102,9 @@ def file_passes(filename, refs, regexs):
     elif extension == "sh":
         p = regexs["shebang"]
         (data, found) = p.subn("", data, 1)
+    elif extension == "py":
```
Comment (spiffxp) [inline hack/boilerplate/boilerplate.py]: pull into above if?
```python
elif extension in ["sh", "py"]:
```

File: hack/boilerplate/boilerplate.py [hunk 2]
```diff
@@ -205,7 +208,7 @@ def get_regexs():
     regexs["date"] = re.compile(get_dates())
     # strip // +build \n\n build constraints
     regexs["go_build_constraints"] = re.compile(r"^(// \+build.*\n)+\n", re.MULTILINE)
-    # strip #!.* from shell scripts
+    # strip #!.* from shell scripts & Python scripts too!
```
Comment (spiffxp) [inline hack/boilerplate/boilerplate.py]:
```suggestion
    # strip #!.* from scripts
```

**Review by spiffxp — CHANGES_REQUESTED** [review]
thanks for doing this, few nits

**Comment by falcon78921** [comment]
@spiffxp Done :)

**Review by spiffxp — APPROVED** [review]

**Comment by falcon78921** [comment]
@spiffxp Thanks again! :)

**Pull request merged**